conpot/protocols/snmp/build_pysnmp_mib_wrapper.py

```diff
@@ -121,15 +165,20 @@ def compile_mib(mib_name, output_dir):
     :param output_dir: Output directory (string).
     """
     # resolve dependencies recursively
-    for dependency in mib_dependency_map[mib_name]:
-        if dependency not in compiled_mibs and dependency in file_map:
-            compile_mib(dependency, output_dir)
+
+    # TODO:
+    # we don't need to solve dependencies ourselves any longer
+    # pysmi handles this automatically
+    #for dependency in mib_dependency_map[mib_name]:
+    #    if dependency not in compiled_mibs and dependency in file_map:
+    #        compile_mib(dependency, output_dir)
     _compile_mib(mib_name, output_dir)
 
 
 def _compile_mib(mib_name, output_dir):
```

Comment:
_compile_mib is only references once throughout the codebase (line 175), therefore a merge will be fine.